docs/dev/podman.md

```diff
@@ -0,0 +1,70 @@
+
+# Podman socket enablement
+
+Clean socket if exists
+--------------------------------------
+```shell
+systemctl disable --now podman.socket
+rm -rf /run/user/${UID}/podman
+rm -rf /run/podman
+```
+
+Open/Enable the socket
+--------------------------------------
+```shell
+systemctl enable --now podman.socket
+systemctl --user enable --now podman.socket
+loginctl enable-linger $USER
+```
+
+Verify it
+--------------------------------------
+```shell
+$ podman -r info | grep remoteSocket -A 2
+  remoteSocket:
+    exists: true
+    path: /run/podman/podman.sock
+```
+
+Enable minikube local registry
+---------------------------------------
+```shell
+minikube addons enable registry 
+```
+Expose the registry to the host machine on port 5000.
+Can be done by xinet port-forward:
+
+###xinet
+Change registry service to a LoadBalancer (minikube tunnel will assign an external IP)
+```shell
+kubectl patch service $REGISTRY_SERVICE_NAME -n $REGISTRY_SERVICE_NAMESPACE --type json -p='[{"op": "replace", "path": "/spec/type", "value":"LoadBalancer"}]'
```

Comment:
replace the values of ``$REGISTRY_SERVICE_NAME`` and ``$REGISTRY_SERVICE_NAMESPACE`` with the hard-coded values minikube addon uses